great-idea-website/css/index.css

```diff
@@ -63,4 +63,200 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+.container {
+    width: 880px;
+    /* TB LR */
+    margin: 0 auto;
+}
+
+/* header */
+
+header.main-header{
+    display: flex;
+}
+
+header nav {
+    width: 691px;
+    vertical-align: top;
+    margin-top: 35px;
+}
+
+header nav a {
+    margin-right: 65px;
+    font-size: 14px;
+    color: gray;
+    text-decoration: none;
+}
+
+header nav a:hover {
+    text-decoration: underline;
+}
+
+header nav a:first-child {
+    margin-left: 0;
+}
+
+header nav a:last-child {
+    margin-right: 0;
+}
+
+header .logo {
+    margin-top: 25px;
+}
+
+/* cta */
+.cta{
+    display: flex;
+}
+
+.cta .cta-text {
+    vertical-align: top;
+    font-size: 77px;
+    margin-top: 120px;
+    text-align: center;
+    padding: 0px 75px;
+}
+
+.cta .cta-text button {
+    background: white;
+    vertical-align: top;
+    margin-top: 6px;
+    padding: 8px 44px;
+    font-size: 17px;
+    border: 1px solid black;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+.cta img {
+    margin-top: 69px;
+}
+
+.main-content {
+    border-top: 1px solid black;
+    margin-top: 76px;
+    padding-top: 39px;
+}
+
+.main-content .top-content {
+    padding: 5% 10px;
+    display: flex;
+}
+
+.main-content .top-content .text-content {
+    width: 45%;
+    margin: 20px 5%;
+    /*display: flex;*/
+    /*display: inline-block;*/
+}
+
+.main-content .middle-img {
+    margin-top: 45px;
+}
+
+.main-content .bottom-content {
+    margin-top: 37px;
+    padding: 0 10px;
+    display: flex;
+}
+
+.bottom-border-1px {
+    border-bottom: 1px solid black
+}
+
+.main-content .bottom-content .text-content {
+    width: 45%;
+    margin: 20px 5%;
+    /*display: inline-block;*/
+    /*display: flex;*/
+}
+
+.main-content .bottom-content .text-content:last-child {
+    margin-right: 0;
+}
+.bottom-content-left {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.styled-ul {
+    list-style: initial;
+    margin: 10px 4%;
+
+}
+.styled-ul li {
+    margin: 5px 0;
+}
+
+.contact {
+    margin-top: 57px;
+    border-top: 1px solid black;
+    padding: 39px 10px 0;
+}
+
+.contact p {
+    margin: 20px 0;
+}
+
+.page-header {
+    margin: 20px 0;
+    /*font-weight: bold;*/
+}
+.page-header h1 {
+    font-size: 24px;
+}
+
+.margin-left{
+    margin-left: 5%;
+}
+
+.flex-wrap {
+    flex-wrap: wrap;
+}
+
+/*noinspection CssInvalidPropertyValue*/
+.flex-space-even {
```

Comment:
these lines are awesome, very clever and a great way to have DRY (Dont Repeat Yourself) code
there are better ways to do this that well learn in the coming weeks, but great job thinking out side of the box!